tests/integration-tests/cfn_stacks_factory.py

```diff
@@ -298,8 +298,14 @@ def delete_all_stacks(self, excluded_stacks=None):
         """Destroy all created stacks except for those in excluded_stacks."""
         logging.debug("Destroying all cfn stacks")
         for value in reversed(OrderedDict(self.__created_stacks).values()):
-            if excluded_stacks and value.name in excluded_stacks:
-                continue
+            if excluded_stacks:
+                excluded = False
+                for stack in excluded_stacks:
+                    if stack in value.name:
```

Comment:
Why is it `stack in value.name`?

i.e. why is it not `value.name in stack` or `stack == value.name`?